build.gradle

```diff
@@ -94,40 +94,40 @@ subprojects { subproject ->
 		derbyVersion = '10.11.1.1'
 		eclipseLinkVersion = '2.4.2'
 		ftpServerVersion = '1.0.6'
-		groovyVersion = '2.4.4'
-		guavaVersion = '18.0'
+		groovyVersion = '2.4.5'
+		guavaVersion = '19.0'
 		hamcrestVersion = '1.3'
 		hazelcastVersion = '3.5.1'
 		hibernateVersion = '4.3.10.Final'
-		hsqldbVersion = '2.3.2'
+		hsqldbVersion = '2.3.3'
 		h2Version = '1.4.180'
-		jackson2Version = '2.5.1'
+		jackson2Version = '2.5.4'
 		javaxActivationVersion = '1.1.1'
-		javaxMailVersion = '1.5.2'
+		javaxMailVersion = '1.5.5'
 		jedisVersion = '2.7.3'
 		jmsApiVersion = '1.1-rev-1'
 		jpa21ApiVersion = '1.0.0.Final'
 		jpaApiVersion = '2.0.0'
-		jrubyVersion = '1.7.19'
+		jrubyVersion = '1.7.23'
 		jschVersion = '0.1.52'
 		jsonpathVersion = '2.1.0'
-		junitVersion = '4.11'
+		junitVersion = '4.12'
```

Comment:
See the several last IO CI failed builds:
```
Caused by: java.lang.IllegalStateException: SpringJUnit4ClassRunner requires JUnit 4.12 or higher.
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.<clinit>(SpringJUnit4ClassRunner.java:98)
	... 36 more
```
So, this upgrades should be merged to "green" the IO CI plan.